pkg/minikube/cluster/cluster.go

```diff
@@ -308,6 +296,7 @@ func MountHost(api libmachine.API, ip net.IP, path, port, mountVersion string, u
 }
 
 // GetVMHostIP gets the ip address to be used for mapping host -> VM and VM -> host
+// TODO: remove this (why not just using driver.GetIP?)
```

Comment:
This is kind of the opposite - driver.GetIP is the IP address of the VM from the host, this is the IP of the host from the VM.

Reply:
OK. I see. I am removing this comment. but i still want to see if it's possible to remove the switch for driver names sometime in future.